src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -6,17 +6,14 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.AspNetCore.Components.Rendering;
-using Microsoft.AspNetCore.Components.Server;
-using Microsoft.AspNetCore.Components.Server.Circuits;
 using Microsoft.AspNetCore.SignalR;
 using Microsoft.Extensions.Internal;
 using Microsoft.Extensions.Logging;
 using Microsoft.JSInterop;
 
-namespace Microsoft.AspNetCore.Components.Web.Rendering
+namespace Microsoft.AspNetCore.Components.Server.Circuits
 {
-    internal class RemoteRenderer : Renderer
+    internal class RemoteRenderer : Microsoft.AspNetCore.Components.RenderTree.Renderer
```

Comment:
Why does this need the fully qualified namespace?

Reply:
Because there are conflicts between the `ArrayBuilder<>` defined in this assembly and defined in M.A.Components.